scico/linop/xray/_xray.py

```diff
@@ -94,14 +95,22 @@ def __init__(
         self.angles = angles
 
         self.nx = np.array(im_shape)
-
-        if x0 is None:
-            x0 = -self.nx / 2
-        self.x0 = x0
         if dx is None:
             dx = np.ones(2)
```

Comment:
Use a default that is less likely to violate the detector bin condition for the projector to work properly?